packages/mapviewer/tests/cypress/tests-e2e/importToolFile.cy.js

```diff
@@ -502,12 +502,26 @@ describe('The Import File Tool', () => {
             fixture: outOfBoundKMLFile,
         })
 
+        const validOnlineUrlWithInvalidContentType = 'https://somes3bucket.com/valid-kml-file.kml'
+        createHeadAndGetIntercepts(
+            validOnlineUrlWithInvalidContentType,
+            'ValidKmlFileWrongContentType',
+            {
+                headers: {
+                    'Content-Type': 'application/octet-stream',
+                },
+                statusCode: 200,
+                body: `<kml> This is an empty kml</kml>>`,
+            }
+        )
```

Comment:
I think you missed that this function has a 4th param to set the `HEAD` config, here you are only setting the `GET`, meaning the `Content-Type` for the `HEAD` will still be valid (coming from the default value for the 4th param)